kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/AlCustomizationSettings.java

```diff
@@ -836,6 +836,10 @@ public boolean getHideAttachmentOptionsWithBots() {
         return hideAttachmentOptionsWithBots;
     }
 
+    public boolean getHideChatBarWithBots() {
+        return hideChatBarWithBots;
```

Comment:
getter for boolean should start from iS, like isHideChatbarWithBots()

Reply:
Done

Reply:
Added in [ace8fb1](https://github.com/Kommunicate-io/Kommunicate-Android-Chat-SDK/pull/551/commits/ace8fb1c1b406d3776cfab6178932cc5f973f1b9)